pkg/resources/resource_cluster.go

```diff
@@ -63,6 +77,16 @@ func clusterCreate(ctx context.Context, d *schema.ResourceData, meta interface{}
 	o := materialize.ObjectSchemaStruct{Name: clusterName}
 	b := materialize.NewClusterBuilder(meta.(*sqlx.DB), o)
 
+	// replication factor
+	if v, ok := d.GetOk("replication_factor"); ok {
+		b.ReplicationFactor(v.(int))
```

Comment:
Nit. We should probably only set replication factor if size is also valid.

Reply:
good catch! just refactored this accordingly